easysid.cpp

```diff
@@ -38,6 +55,35 @@ EXPORT void end_capture()
     }
 }
 
+EXPORT void set_chip(uint32_t chip)
+{
+    if (chip < 0 || chip > 1) {
+        // TODO: Implement errors, or SEH to handle these scenarios.
+        return;
+    }
+    sid.set_chip_model(static_cast<chip_model>(chip));
+}
+
+EXPORT void set_sample_parameters(double clock, uint32_t sample_mode, uint32_t sample_freq)
```

Comment:
The corresponding function name in resid is "set_sampling_parameters". Is there a reason that you've changed the name here to "set_sample_parameters"?